src/rpdk/core/fragment/generator.py

```diff
@@ -91,24 +77,7 @@ def __validate_no_exports_present(raw_fragments):
                         "Found an Export statement in Output: " + _output_name
                     )
 
-    def __validate_output_limit(self, raw_fragments):
-        if "Outputs" in raw_fragments:
-            output_count = len(raw_fragments["Outputs"].items())
-            if output_count > self.output_limit:
-                raise FragmentValidationError(
-                    "The Module template fragment has "
-                    + str(output_count)
-                    + " outputs but must not exceed the limit of "
-                    + str(self.output_limit)
-                    + " outputs"
-                )
-
     def __validate_resources(self, raw_fragments):
-        if "Resources" not in raw_fragments:
```

Comment:
Validations like this have been added to avoid parsing errors that are impossible to understand by the user.  I'm not sure if we're regressing on the user experience here.

Reply:
`cfn-lint` will throw `E1001 Missing top level template section Resources` for this validation

Reply:
@PatMyron For all the validations that you removed in this PR can you show us what it looks like right now and what it will look like after? Wanna make sure that the error messages in cfn-lint are on par with the ones in the CLI